README.md

```diff
@@ -21,7 +21,74 @@ This is a Lambda Labs Capstone Project designed for rental property management.
+	* It allows for reusability of already written components which reduces development time
+	* It is a tried and tested framework that will help us avoid sudden shifts in the technology causing major refactors
+
+#### React Hooks
+
+React Hooks simplify working with React due to use near exclusive use of functional components versus class components. Also, the use of React hooks results in smaller package sizes.
+
+#### Typescript
+
+	* Stricter type-checking will speed up development by reducing time spent on finding bugs and limiting the number of bugs introduced in the first place.
+	* Explicit type declarations will communicate to other developers the intent and use of a given piece of code.
+  * TypeScript integrates very well with webpack and babel, both of which are essential tools for using modern JS features.
+
+### Frontend deployed to Netlify
+
+### Backend built using:
+
+#### Node.js, Express.js
+
+	* Node.js offers a unified language for both front-end and back-end, giving all team members flexibility to work on either when necessary.
+	* Express.js allows us to code and customize the back-end to our liking, and gives us more control on what and how the back-end handles requests.
+	* There is very detailed documentation available for each.
+	* There also widespread community support for Node and Express.
+
+#### PostgreSQL 
+
+	* Saves the necessary data in an efficient way
+	* Allows for easy queries through JOIN support
+	* Allows for very easy deployment
+	* Has a great community around it and is future proof / very independent of new trends
+
+### Backend deployed to Heroku
+
+## Authentication
+
+App users are authenticated using JSON web tokens (JWT).
+
+## Testing
+
+This application was tested at every stage of development by each contributor. We used Husky to automate unit testing and run Prettier linting on each commit. This ensured that we could continuously deploy to Netlify and Heroku without the risk of breaking our build. All pull requests were reviewed by at least one other team member and merged by a third party to ensure that we had as many eyes on the code as possible searching for potential bugs.
+
+## Installation Instructions
+
+To install this app, make sure you have node.js and npm installed. From the root directory, run npm install or yarn to install dependencies for frontend and backend.
+
+Start the development server by running npm dev:server or yarn dev:server.
+
+### Other Scripts
+
+	* yarn typecheck - runs the TypeScript compiler
+	* yarn build - creates a build of the application
+	* yarn start - starts the production server after a build is created
+	* yarn test - runs tests in __tests__ directory
+
+## Stripe
+
+All user payments are processed using Stripe via the react-stripe-elements plugin. Once the user submits their payment information, it is converted to a token that can be safely sent to the server. This eliminates the need for our app to handle sensitive credit card data.
```

Comment:
I would mention here briefly that using Stripe `minimizes the business's financial and legal liability` or something to this effect.